### PR TITLE
#545 Singapore MRT under construction and Sentosa Express line style

### PR DESCRIPTION
**Pull request by thekingofcity**

### CODE REVIEW

File: src/components/svgs/lines/styles/mrt-under-construction.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    );
+};
+
+/**
+ * MRTUnderConstruction specific props.
+ */
+export interface MRTUnderConstructionAttributes extends LinePathAttributes, AttributesWithColor {}
+
+const defaultMRTUnderConstructionAttributes: MRTUnderConstructionAttributes = {
+    color: [CityCode.Beijing, 'bj1', '#c23a30', MonoColour.white],
```
Comment (langonginc) [inline src/components/svgs/lines/styles/mrt-under-construction.tsx:46]: Maybe Singapore/ewl.

**Review by langonginc — CHANGES_REQUESTED** [review]

File: src/components/svgs/lines/styles/mrt-under-construction.tsx (hunk 1 above)
Comment (thekingofcity) [reply src/components/svgs/lines/styles/mrt-under-construction.tsx]: Speaking of automatically overwriting the runtime color to the elements' color, I do have issue #478 for it. But the question is how do we provide a switch for users as both approaches are needed in different cases.

Comment (langonginc) [reply src/components/svgs/lines/styles/mrt-under-construction.tsx]: A check box beside 'Color' in the side bar?

If we do this, we can add a select box to choose line style type.

**Review by langonginc — APPROVED** [review]

File: src/components/svgs/lines/styles/mrt-under-construction.tsx (hunk 1 above)
Comment (thekingofcity) [reply src/components/svgs/lines/styles/mrt-under-construction.tsx]: Or some tweaks in settings. Not sure what the percentage of users who would like to overwrite or use the default color.

Having much on the tools panel may make it hard to use. Users may lost on so many options and we could leave some advanced/professional features in places not easy to find.

Comment (thekingofcity) [reply src/components/svgs/lines/styles/mrt-under-construction.tsx]: Let's move more replies to #478 and merge this now :)

**Pull request merged**